cli/cli.go

```diff
@@ -144,6 +149,10 @@ func (h *HostInput) Set(value string) error {
 	return nil
 }
 
+func (h *HostInput) Type() string {
```

Comment:
Please add documentation to this function. It would be good to say why this is needed.